components/arm/client.go

```diff
@@ -13,6 +13,7 @@ import (
 	rprotoutils "go.viam.com/rdk/protoutils"
 	"go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/robot/framesystem"
```

Comment:
I don't like the addition of a dependency on the framesystem.  Lets move the `ParseKinematicsResponse` to the `referenceframe` package and rename to something like `KinematicModelFromProtobuf` or `NewModelFromProtobuf`

Reply:
the urdf model depends on referenceframe.

I can't move `KinematicModelFromProtobuf` into `referenceframe` without causing a cycle.